spec/features/schemes_helpers.rb

```diff
@@ -68,7 +68,7 @@ def fill_in_and_save_location
   end
 
   def fill_in_and_save_second_location
-    fill_in "Postcode", with: "XX1 1XX"
+    fill_in "Postcode", with: "AA1 2AA"
```

Comment:
Maybe worth flipping this so that we enter a postcode with no spaces and it saves and formats it it spaces?

Reply:
Not a bad idea, the issue here was that invalid postcodes are not parsed properly (returns the same input string).